src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -21,13 +21,13 @@ export default dedupingMixin(
 
         let ripple = null;
         const rippleWrapper = document.createElement("div");
-        this.shadowRoot.appendChild(rippleWrapper);
+        this.parentElement.appendChild(rippleWrapper);
         Object.assign(rippleWrapper.style, {
           position: "absolute",
           width: isTouch ? "100px" : "50px",
           height: isTouch ? "100px" : "50px",
-          top: "50%",
-          left: "50%",
+          top: this.style.top,
```

Comment:
I assumed it was, but it turns out it isn't...

```javascript
top: this.style ? this.style.top : false || "0";
```
works, but as you say, It could just as well be positioned using `bottom` or `right`. And also, if someone sets a different `position` it would mess things up as well...

Perhaps the div needs to be repositioned when the click happens...

Or a div that floats on top of the entire page and moves below the mouse on mousedown... that would be easier to port to other things than elements as well.